deps/exokit-bindings/webaudiocontext/src/AudioContext.cpp

```diff
@@ -247,6 +244,17 @@ NAN_METHOD(AudioContext::New) {
   AudioContext *audioContext = new AudioContext(sampleRate);
   audioContext->Wrap(audioContextObj);
 
+  if (!threadInitialized) {
+    uv_timer_init(uv_default_loop(), &timer_req);
```

Comment:
I get that we want to run `runInMainThread` in the main thread, but why every 100ms instead of using something like [`uv_async`](http://docs.libuv.org/en/v1.x/async.html) at the exact moment we need it (e.g. there is something in queue)?

That seems it would be not only cleaner but more responsive -- I don't think we can afford to wait any ms for streaming audio, if my reading of this code is correct.

Reply:
Also, I think `uv_default_loop` might not be a correct assumption with workers, which have their own loop. It might need to be be the "current" loop from `child-process-thread`/`window-worker`.

This isn't terrible, it just might preclude WebAudio in workers.